models/text_detection_ppocr/demo.cpp

```diff
@@ -172,7 +172,10 @@ int main(int argc, char** argv)
                 imwrite("result.jpg", originalImage);
             }
             if (viz)
+            {
                 imshow(kWinName, originalImage);
+                waitKey(0);
```

Comment:
Line 30 has
```c++
 while (waitKey(1) < 0)
```
So I think we don‘t need to add `waitKey(0)`?

Reply:
`while (waitKey(1) < 0)` - I believe this has a timeout of 1 ms before continuing the loop.

It then attempts to go into 
```
cap >> originalImage;
if (originalImage.empty())
{
    if (parser.has("input"))
    {
        cout << "Frame is empty" << endl;
        break;
    }
    else
        continue;
}
```
and reaches `cout << "Frame is empty" << endl;`. On the otherhand, waitKey(0) waits indefinitely, allowing the user to keep the processed image up for viewing.